src/main/config/config_eeprom.c

```diff
@@ -147,19 +126,18 @@ bool isEEPROMContentValid(void)
             return false;
         }
 
-        chk = updateChecksum(chk, p, record->size);
+        crc = crc16_ccitt_update(crc, p, record->size);
 
         p += record->size;
     }
 
     const configFooter_t *footer = (const configFooter_t *)p;
-    chk = updateChecksum(chk, footer, sizeof(*footer));
+    crc = crc16_ccitt_update(crc, footer, sizeof(*footer));
     p += sizeof(*footer);
-    chk = ~chk;
-    const uint8_t checkSum = *p;
+    const uint16_t checkSum = *(uint16_t *)p;
```

Comment:
@martinbudden : The resulting value should be 0x1D0F is CRC is stored inverted in message. Thi value may be #defined together with CRC_START_VALUE
```
crc = ~ crc;
config_streamer_write(&streamer, (uint8_t *)&crc, sizeof(crc));
```

If it is stored directly, CRC of whole message will be zero.